certified-connectors/Zippydoc/apiDefinition.swagger.json

```diff
@@ -0,0 +1,331 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ZIPPYDOC",
+    "description": "ZIPPYDOC is a platform that allows to merge, transform, filter, extract, report and analyze structured and table based data and to flexibly create data flows",
```

Comment:
@worm333 could you please change the description just a little bit? It should be concise and reflect the main value and purpose offered by your connector.
More information about the description can be found [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector).

Reply:
Hi @v-EgorKozhadei , I've updated description, so it describes in more detail the main purpose and value offered by ZIPPYDOC